tox.ini

```diff
@@ -17,8 +17,8 @@ skip_install = true
 envdir = {toxworkdir}/py36
 commands=
     ; Force packaging even if setup.{py,cfg} haven't changed
-    rm -rf ./*.egg-info/
-    python setup.py coverage
+    ;rm -rf ./*.egg-info/
+    ;python setup.py coverage
```

Comment:
Luis, looks like this was meant to be a temporary change, right? As it is, it's bypassing the tests that `coverage` used to run, could you double check this?